test/Microsoft.ML.Benchmarks/RffTransform.cs

```diff
@@ -43,7 +43,7 @@ public void CV_Multiclass_Digits_RffTransform_OVAAveragedPerceptron()
             var pipeline = mlContext.Transforms.Projection.CreateRandomFourierFeatures("Features", "FeaturesRFF")
             .AppendCacheCheckpoint(mlContext)
             .Append(mlContext.Transforms.Concatenate("Features", "FeaturesRFF"))
-            .Append(new ValueToKeyMappingEstimator(mlContext, "Label"))
+            .Append(new ValueToKeyMappingEstimator(mlContext, "Label", "Label"))
```

Comment:
>, "Label" [](start = 69, length = 9)

Now that the `source` column has a default value again, from what I see, we can get rid of this, right?